config/extensions.neon

```diff
@@ -90,3 +90,7 @@ services:
         class: staabm\PHPStanDba\Ast\PreviousConnectingVisitor
         tags:
         	- phpstan.parser.richParserNodeVisitor
+    -
+        class: staabm\PHPStanDba\Extensions\DibiConnectionFetchDynamicReturnTypeExtension
```

Comment:
please separate the phpstan extensions (and reflection) related stuff into a separate PR.
we will come back to them after we landed syntax error detection first.

Reply:
removed